docs/src/index.rst

```diff
@@ -0,0 +1,23 @@
+*****************************************************************
+A Getting Started Guide to the Genomic Epidemiology of SARS-CoV-2
+*****************************************************************
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
I think this should be a `maxdepth` of 2, so each of the subsections below include links to their pages. The current rendering (see the screenshot in the `self` comment) only lists links to the analysis, visualization, and reference sections when these could be lists of links to specific pages. The result should be a ToC that is more like [the current ncov tutorial index](https://nextstrain.github.io/ncov/).

Reply:
👍 I really like the GitHub pages layout, and find it generally preferable to our RTD setup. Specifically, 
- I like "sections" (e.g. "Analysis") to be simply headings in the sidebar, not pages in and of themselves.
- I like seeing all the page titles of each section in the sidebar at once
- As the number of pages & sections grow, it becomes appropriate to expand / contract the listing of pages in each section, as there are simply too many to show all at once. Currently, RTD often reloads the page / jumps to another page when all I want to do is expand that section. I really like the behavior of [the react docs](https://reactjs.org/docs/getting-started.html) in this case.